jwst/cube_build/instrument_defaults.py

```diff
@@ -494,47 +494,26 @@ def SetSpatialSize(self, value, parameter1, parameter2=None):
             self.Info[parameter1][parameter2]['bscale'] = value
 
 
-    def SetSpectralStep(self, value, parameter1, parameter2=None):
-        if parameter2 is None: # data is NIRSPEC paramters do not vary with filter
-            self.Info[parameter1]['wscale'] = value
-        else:
-            self.Info[parameter1][parameter2]['wscale'] = value
+    def SetSpectralStep(self, value, parameter1, parameter2):
+        self.Info[parameter1][parameter2]['wscale'] = value
```

Comment:
`self.Info` should be `self.info` as it is a dict, not a class.  Not necessary for this PR, but just an observation.

Reply:
Similarly, these `SetSpectralStep` functions should be `set_spectral_step`, as they are methods, not classes.  Again, not necessary change for this PR, but to get things more [PEP8 compliant](https://www.python.org/dev/peps/pep-0008/?#naming-conventions).

Reply:
ok I will add that to the list of things to change after Build 7.2